backend/routers/spiritual.py

```
@@ -789,4 +907,136 @@ def get_default_lord(house_index: int) -> str:
+        if user_data['birth_chart_data']:
+            try:
+                chart_data = json.loads(user_data['birth_chart_data']) if isinstance(user_data['birth_chart_data'], str) else user_data['birth_chart_data']
+                # Check if data is recent (less than 30 days old)
+                if chart_data.get('cached_at'):
+                    cached_date = datetime.fromisoformat(chart_data['cached_at'].replace('Z', '+00:00'))
+                    if (datetime.now() - cached_date).days < 30:
+                        return {
+                            "success": True,
+                            "complete_profile": chart_data
+                        }
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)